Sources/Jetworking/Interceptor/Request/LoggingRequestInterceptor.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+public final class LoggingRequestInterceptor: RequestInterceptor {
```

Comment:
I suggest to merge LoggingRequestInterceptor and LoggingResponseInterceptor into a LoggingInterceptor like Moya is doing. In most cases we want to log requests and responses together.

Reply:
Ok I realized that this is not possible yet since we separate the Request and Response interceptors. Maybe we could find a solution to merge them together that it doesn't matter if it is for requests or responses. I will create an issue for this.

Reply:
Possible solution could be to have another option in the client configuration like `isLoggingEnabled` which then implicitly adds the logging interceptors when active. 
What do you think about that?

Reply:
Will be discussed in a future ticket